### PR TITLE
Devops

### PR DESCRIPTION
**Pull request by acozianu**

### CODE REVIEW

File: src/services/TestResultsService.js [hunk 1]
```diff
@@ -16,64 +17,64 @@ class TestResultsService {
...
+    return data.Items
   }
 
-  filterTestResultByDate (testResults, fromDateTime, toDateTime) {
-    for (let i = 0; i < testResults.length; i++) {
-      testResults[i].testTypes = testResults[i].testTypes.filter(
-        function (testType) {
-          return (!(dateFns.isAfter(testType.createdAt, toDateTime) || dateFns.isBefore(testType.createdAt, fromDateTime)))
-        })
+  checkDAOResponse (data, filters) {
```
Comment (adpetrescu) [inline src/services/TestResultsService.js]: I think this function should be renamed to something like `filterTestResults` / `applyTestResultsFilters`

File: src/services/TestResultsService.js [hunk 2]
```diff
@@ -16,64 +17,64 @@ class TestResultsService {
...
     }
-    testResults = testResults.filter(
-      function (testResult) {
-        return testResult.testTypes.length !== 0
-      })
+    if (filters.testStationPNumber) {
+      testResults = GetTestResults.filterTestResultsByParam(testResults, 'testStationPNumber', filters.testStationPNumber)
+    }
+    if (testResults.length === 0) {
+      return new HTTPError(404, 'No resources match the search criteria')
```
Comment (adpetrescu) [inline src/services/TestResultsService.js]: Shouldn't this error be caught in `getTestResults` and also thrown?

File: serverless.yml [hunk 3]
```diff
@@ -44,6 +44,13 @@ functions:
               status: false
               fromDateTime: false
               toDateTime: false
+  getTestResultsByTesterStaffId:
+    handler: src/functions/getTestResultsByTesterStaffId.getTestResultsByTesterStaffId
+    events:
```
Comment (adpetrescu) [inline serverless.yml]: Are the events needed if we use the offline lambda invoke library?

File: src/services/TestResultsService.js (hunk 1 above)
Comment (acozianu) [reply src/services/TestResultsService.js]: done

File: serverless.yml (hunk 3 above)
Comment (acozianu) [reply serverless.yml]: I am not sure. Let's keep it like this for the moment.

File: src/services/TestResultsService.js (hunk 2 above)
Comment (acozianu) [reply src/services/TestResultsService.js]: Done

**Pull request merged**